SimpleTimer.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using SubsurfaceStudios.Attributes;
+using SubsurfaceStudios.Utilities.Async;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace SubsurfaceStudios.Utilities.Time
```

Comment:
In the future (not now) I'd like to change this namespace, as `Time` often conflicts with the `UnityEngine.Time` static class. Not important now, as we use it elsewhere, but I think it would be worthwhile to be aware of.